openhtf/core/measurements.py

```diff
@@ -139,6 +140,19 @@ def _initialize_value(self):
     else:
       self.measured_value = MeasuredValue(self.name)
 
+  def __deepcopy__(self, memo):
```

Comment:
If we're going to side-step the ValueError there, then we might as well get rid of that check all together.  I'd be in favor of removal, but I assumed it was there for a reason.